BHoM_UI/Global/SearchMenu.cs

```diff
@@ -92,7 +92,6 @@ protected virtual List<SearchItem> GetAllPossibleItems()
                 new SearchItem { Item = typeof(MoveCaller).GetMethod("Move"), CallerType = typeof(MoveCaller), Icon = Properties.Resources.Move },
                 new SearchItem { Item = typeof(PullCaller).GetMethod("Pull"), CallerType = typeof(PullCaller), Icon = Properties.Resources.Pull },
                 new SearchItem { Item = typeof(PushCaller).GetMethod("Push"), CallerType = typeof(PushCaller), Icon = Properties.Resources.Push },
-                new SearchItem { Item = typeof(UpdatePropertyCaller).GetMethod("UpdateProperty"), CallerType = typeof(UpdatePropertyCaller), Icon = Properties.Resources.UpdateProperty },
```

Comment:
This is clashing with an pre-existing PR: https://github.com/BHoM/BHoM_UI/pull/147